.travis.yml

```diff
@@ -2,9 +2,7 @@ sudo: false
 
 language: node_js
 node_js:
-  - "4"
   - "6"
-  - "stable"
```

Comment:
@rwjblue was there a reason why we ran this on all supported Node versions? 🤔

Reply:
Don’t think so. In general we mostly use “oldest” supported node for addons these days, so I figured this was probably fine.